doc/integrator/install_application.rst

```diff
@@ -165,6 +167,23 @@ using a C compiler.
 Furthermore, some changes in the apache wsgi and mapserver configurations are
 required to make c2cgeoportal work on Windows.
 
+Also, between all the different command interfaces available on windows (cmd, 
+Cygwin, git mingw), only Windows default cmd interface handle paths correctly
+in all stage of the application setup.
+
+Command interface and environment variable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Only use Windows default commande interface::
+
+    Start > Run... > cmd
+
+Cygwin and git mingw are not compatible. Powershell is untested.
+
+Complementarily you need to add all the ressource paths to your system PATH
+environment variable, for cygwin, git, node, apache, postgres, mapserver and
```

Comment:
Hmmm... Why do we need Mapserver, Apache, PostgreSQL and Proj / nad in the path ???

We always try to keep the path definition so little as possible... because in Windows things can quickly get mixed up and terribly wrong...